controller/src/main/java/io/pravega/controller/store/stream/PersistentStreamBase.java

```diff
@@ -291,19 +306,65 @@ public String getScopeName() {
      *
      * @param newRanges      key ranges of new segments to be created
      * @param scaleTimestamp scaling timestamp
+     * @param runOnlyIfStarted run only if the scale operation was started.
```

Comment:
This is described in the description of streamMetadataTasks startScale.
For this methods purpose, it will perform the requested processing only if this param was true and processing was already started.